### PR TITLE
Bugfix for output writer

### PR DESCRIPTION
**Pull request by babrodtk**
Somehow rsSat/rvSat were never set in original PR, and this should fix that issue. 

Related to issue OPM/opm-output#88

### CODE REVIEW

File: opm/autodiff/BlackoilModelBase.hpp [hunk 1]
```diff
@@ -323,7 +323,7 @@ namespace Opm {
         bool use_threshold_pressure_;
         V threshold_pressures_by_connection_;
 
-        SimulatorData sd_;
+        mutable SimulatorData sd_;
```
Comment (babrodtk) [inline opm/autodiff/BlackoilModelBase.hpp]: rsSat/rvSat is set/updated within variableStateExtractVars(...) const, which this is made mutable

**Comment by bska** [comment]
jenkins run polymer norne spe please

**Comment by blattms** [comment]
Great! This fixes the exception for model 2 for me.

I still have no idea why it was not an issue for other models.

**Comment by atgeirr** [comment]
Thanks for patch, review and testing! Merging.

**Pull request merged**